src/sparseml/pytorch/torchvision/train.py

```diff
@@ -444,6 +444,10 @@ def collate_fn(batch):
         return
 
     manager = ScheduledModifierManager.from_yaml(args.recipe_path)
+    if args.one_shot:
+        manager.apply(model)
+        return
```

Comment:
What about saving the one shot checkpoint as in code block at line 476 - line 492

Reply:
Added